packages/capesolo.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,8 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'CAPEsolo'
+$category = 'Debuggers'
```

Comment:
I am not sure if `Debuggers` is the best category as it does more things. @kevoreilly, @enzok can you confirm that this is the best match from https://github.com/mandiant/VM-Packages/blob/main/categories.txt?

We are ready to merge the addition of capesole after this is discussed. Thanks for the assistance adding the tool and sorry that the review took so long (I was not able to do much testing while working on https://github.com/mandiant/flare-vm/pull/660 as it keeps busy my VirtualBox).

Reply:
I am going to merge this PR to avoid delaying the addition of capesolo. We can change the category later on (@kevoreilly, @enzok please just open a new issue if you think the category should be changed). :blush: